drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -820,7 +820,7 @@ void RasterizerCanvasGLES3::_canvas_item_render_commands(Item *p_item, Item *cur
 
 				RasterizerStorageGLES3::Texture *texture = _bind_canvas_texture(mesh->texture, mesh->normal_map);
 
-				if (texture) {
+				if (texture && texture->width != 0 && texture->height != 0) {
```

Comment:
There are lots of other occurrences of the same code in this file which also need to be handled.

I don't know if this would negatively impact performance, maybe we'd better ensure that textures can't have null widths/heights? CC @clayjohn

Reply:
This one is the last change I'm not 100% sure about, so I'd suggest to move it to a separate PR to be discussed further. Then this PR can be merged.